index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="desk.css">
+    <title>Portfolio</title>
+</head>
+
+<body>
+    <nav id="navbar">
+        <ul id="list">
+            <li><a id="home" href="index.html">Home</a></li>
+            <li><a id="portfolio" href="portfolio.html">Portfolio</a></li>
+            <li><a id="about" href="about.html">About</a></li>
+            <li><a id="contact" href="contact.html">Contact</a></li>
+            <li><img id="line" src="./images/Rectangle_52.png" alt="rectangle to underline options"></li>
+        </ul>
+    </nav>
+
+    <main>
+        <section id="presentation">
+            <article>
+                <p id="name">Francisco Borja</p>
+                <p id="phrase">Hello! I am a software developer! <wbr>I can help you build a product, <br>feature or
+                    website. <wbr>Take a look of my works.<br>
+                    If you like what you see and have a project <wbr>you need coded, <br>don’t hesitate and contact me.
+                </p>
+            </article>
+            <div>
+                <p id="txtbutton">Start collaboration</p>
+            </div>
```

Comment:
- [ ] The button is a link `<a>` or `<button>` tag, and follow the button styling: Active, Hover, and Pressed ([see design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=67%3A1095))

Reply:
not resolved